input_writer.go

```diff
@@ -0,0 +1,30 @@
+package logrus
+
+import (
+	"bufio"
+	"io"
+	"runtime"
+)
+
+func (logger *Logger) InputWriter() (*io.PipeWriter) {
```

Comment:
hum makes sense

@Sirupsen is the owner, I'm just helping out with PRs and issues =]